jmclient/jmclient/wallet.py

```diff
@@ -889,6 +916,16 @@ def yield_imported_paths(self, mixdepth):
         """
         return iter([])
 
+    def is_standard_wallet_script(self, path):
+        """
+        Check if the priv/pub key pair referenced by path is of the same
```

Comment:
Still nit on this comment: the priv/pub keypair itself is not intrinsically anything, only the scriptPubKey constructed from it, is.

Reply:
fixed in 8c3ae11